src/main/java/org/ihtsdo/buildcloud/core/entity/Build.java

```diff
@@ -107,7 +114,7 @@ public Build(final String creationTime, final String productBusinessKey, final B
 	}
 
 	public Build(final Date creationTime, final Product product) {
-		this(EntityHelper.formatAsIsoDateTime(creationTime), product.getBusinessKey(), product.getBuildConfiguration(), product.getQaTestConfig(), PENDING.name());
+		this(EntityHelper.formatAsIsoDateTime(creationTime), product.getReleaseCenter().getBusinessKey(), product.getBusinessKey(), product.getBuildConfiguration(), product.getQaTestConfig(), PENDING.name());
```

Comment:
@CoderMChu But we need a proper product when we create a build, do not we? 

Also this constructor is used in 10 different places:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/20746761/175343603-7985c3a8-e2ba-47a3-bbe0-8c82a074b8fa.png">

Reply:
@astro-snail We should remove this constructor that using product. We should use the product key, buildConfig and QAConfig .. constructor instead so that it will reduce confusions.

Reply:
@CoderMChu got it. I was thinking that we need to remove product altogether. Will remove this constructor. Thank you.